src/components/Bullet.tsx

```diff
@@ -264,7 +271,7 @@ const Bullet = ({
         paddingBottom: extendClickHeight + 2,
         position: 'absolute',
         verticalAlign: 'top',
-        width: 4, // make the bullet wide enough to be clicked, but not enough to encroach on the editable
+        width: -((fontSize - 9) * 0.5 - 11 - extendClickWidth - (isTableCol1 ? fontSize / 4 : 0)) - extendClickWidth, // make the bullet wide enough to be clicked, but not enough to encroach on the editable
```

Comment:
Please factor out the common calculations between `marginLeft` and `width` rather than duplicating code. Duplicating code like this is redundant, unperformant, hard to read, and fragile. It gives the illusion that they are independent, when they are actually tightly coupled. Changing one without changing the other will cause a break, and vice versa.

Reply:
Please remove the comment, as it is no longer relevant.

Reply:
Right, sure sorry about that it skipped my mind, I have fixed this now.